oaknational/Oak-Web-Application
# [PUPIL-789] Pupil nav on 404 and 500

## Description

Music year: 2011

Ticket [here](https://www.notion.so/404-500-Pupil-Navigation-2f96de8298774e7ab2799dc10062428e?pvs=23)

## How to test

Update - Just realised that all the preview builds currently have an issue with the 400 and 500 failing. Making this PR hard to test.


1. Go to https://deploy-preview-2517--oak-web-application.netlify.thenational.academy/pupils/does-not-exist
2. You should see a 404 with the pupil area selected

_

1. Go to homepage and check teachers is selected

_

1. Go to /pupils/years and check pupils is selected

NOTE: You may see teacher navigation initially selected, before it moves to pupil. This was unavoidable without adding server-side rendering to our 404 and 500 pages, which I don't believe was warranted as it may cause a larger issue in a large outage. 

## Checklist
